docs/custom-rules/README.md

```diff
@@ -0,0 +1,123 @@
+# Custom Rules
+
+## How to add custom rules
+
+In order to add your custom rules, first you have to install our project as a devDependency with :
+
+`npm i -DE {{ packageName }}`
+
+Then you need to create a folder with the name of your choice, containing another folder named 'rules'.
+In this 'rules' folder, you can create your first rule. You need to write them using TypeScript as we use decorators to register rules into our application.
```

Comment:
You are able to use decorator without TypeScript. https://babeljs.io/docs/en/babel-plugin-proposal-decorators